usersimcrs/simulator/moviebot/moviebot_rl_sim.py

```diff
@@ -0,0 +1,422 @@
+"""User simulator to interact with MovieBot agent during reinforcement
```

Comment:
Can you expand it with a high-level description of how the simulator works (how goal/info need is generated and how agenda updates work)?

Reply:
Done.